scripts/standard_configure_MINGW.sh

```diff
@@ -40,6 +40,7 @@ add_app ${KRATOS_APP_DIR}/ConvectionDiffusionApplication
 cmake ..                                                                                            \
 -G "MinGW Makefiles"                                                                                \
 -DWIN32=TRUE                                                                                        \
+-DCMAKE_INSTALL_PREFIX="${KRATOS_SOURCE}/bin/${KRATOS_BUILD_TYPE}"                                  \

```

Comment:
This variable with this value should be automatically set by CMake

Reply:
For some reason it is not

Reply:
:+1: I'll take a look